thoth/common/openshift.py

```diff
@@ -858,6 +860,12 @@ def schedule_inspection(
             "memory"
         ]
 
+        if send_messages:
+            template_parameters["SEND_MESSAGES"] = str(send_messages)
+
+        if force_sync:
+            template_parameters["FORCE_SYNC"] = str(force_sync)
```

Comment:
Can we prefix these with `THOTH_`?

Reply:
Yeah, I'll change them here and in the linked PR